### PR TITLE
Blacklist capability

### PR DESCRIPTION
**Pull request by brianedmonds90**
Terminal modifications for blacklist capabilities

### CODE REVIEW

File: app/terminal/terminal.py [hunk 1]
```diff
@@ -100,16 +101,23 @@ def do_st(self, op_id):
         service = self._locals['services']['operation_svc']
         asyncio.run_coroutine_threadsafe(service.run(op_id), loop=self._loop)
         self._sout.write(str(colored('Started!', 'yellow')) + '\n')
+	
+	def add_facts_from_csv(self, csv_path, table_name, op_id):
```
Comment (privateducky) [inline app/terminal/terminal.py]: let's rename this with a _ prefix, to make it more visible that it is private. then move it to the bottom, where other private functions live

File: app/terminal/terminal.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ def do_help(self):
         self._sout.write('\t'+'qu: queue an operation to be started later (required) operation name, adversary ID and group ID (optional) jitter fraction' + '\n')
         self._sout.write('\t'+'re: view a specific result (required) link ID - from the decision chain.' + '\n')
         self._sout.write('\t'+'st: start an operation. (required) a queued operation ID.' + '\n')
+		self._sout.write('\t'+'bl: Import blacklist of facts via a csv file with columns [fact, value, score]. Command usage: bl [csv file path] [op_id]' + '\n')
```
Comment (privateducky) [inline app/terminal/terminal.py]: remove this line - this is not a runnable function

File: app/terminal/terminal.py (hunk 1 above)
Comment (brianedmonds90) [reply app/terminal/terminal.py]: This should be done in the latest commit

File: app/terminal/terminal.py (hunk 2 above)
Comment (brianedmonds90) [reply app/terminal/terminal.py]: Done

File: .gitmodules [hunk 3]
```diff
@@ -13,3 +13,6 @@
 [submodule "plugins/stockpile"]
```
Comment (privateducky) [inline .gitmodules]: remove this change - no new plugins required.

File: app/terminal/terminal.py [hunk 4]
```diff
@@ -128,6 +121,16 @@ def _flush(self, content):
         content = colored(content, 'yellow')
         self._sout.write(str(content) + '\n')
 
+    def _add_facts_from_csv(self, csv_path, op_id):
```
Comment (privateducky) [inline app/terminal/terminal.py]: since we're back to a single function, let's put this _add_facts_from_csv logic back under the do_up function

**Pull request merged**